rubocop_baseline.yml

```diff
@@ -0,0 +1,520 @@
+---
+require:
+  - rubocop-performance
+  - rubocop-rspec
+AllCops:
+  TargetRubyVersion: '2.6'
```

Comment:
Why set this to 2.6 while the gemspec is 2.7+?
https://github.com/puppetlabs/puppet-lint/blob/2e0087d801383c59a27c01f351375132d0b20830/puppet-lint.gemspec#L35
I thought Rubocop even raises a warning about that.

Reply:
Resolved.